Azure/communication-ui-library-ios
# [Feature] Implement message send status (Sending, delivered, failed)

## Purpose

* Implemented sending, delivered and failed status icon

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull request checklist

This PR has considered:

## How to Test


* Open a chat thread
* Send out a message without all other participants viewed it.
* Sending icon should appear shortly on the last message and then be replaced by sent icon  

* Turn off wi-fi, then send out a message. Failed icon should appear on every failed messages

* The last seen message and the last sending/sent message doesn't have to be the same message


## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |
<img src="https://user-images.githubusercontent.com/107075081/202996019-c95e76ed-46c9-4482-9fac-fcd870ac75e3.png" width="200" height="400" /> | <img src="https://user-images.githubusercontent.com/107075081/205704684-10dd9502-b17c-4090-8ec3-b1b6aafff61a.PNG" width="200" height="400" />

## Other Information
